resources/testdriver.js

```diff
@@ -78,9 +78,17 @@
          * Trigger user interaction in order to grant additional privileges to
          * a provided function.
          *
-         * https://html.spec.whatwg.org/#triggered-by-user-activation
+         * See `triggered by user activation
+         * <https://html.spec.whatwg.org/#triggered-by-user-activation>`_.
```

Comment:
What's the trailing underscore doing here?

Reply:
Hmm, there's plenty of this, so it's probably a supported syntax that does something useful.